components/lsp-form/LongShort.tsx

```diff
@@ -31,22 +40,40 @@ const LongShort: FC<Props> = ({
           placeholder="0.0"
           value={longTokenAmount}
           setValue={setLongTokenAmount}
+          additionalEffects={(e) => {
+            // Note: There is an effect that will cause the other input to update.
+            const newAmount =
+              Number(e.target.value) * Number(collateralPerPair);
+            setAmount(newAmount.toString());
+          }}
         />
         <TextInput
           label="short token"
           labelPlacement={"overlap" as LabelPlacement}
           placeholder="0.0"
           value={shortTokenAmount}
           setValue={setShortTokenAmount}
+          additionalEffects={(e) => {
+            // Note: There is an effect that will cause the other input to update.
+            const newAmount =
+              Number(e.target.value) * Number(collateralPerPair);
+            setAmount(newAmount.toString());
+          }}
         />
       </FormRow>
       <BalanceRowToken>
         <div>
-          <span>Your Balance 123.45 </span>
+          <span>
+            Your Balance{" "}
+            {ethers.utils.formatUnits(tokensMinted.toString(), erc20Decimals)}{" "}
+          </span>
           {!collateralOnTop && redeemForm && <span>Max</span>}
         </div>
         <div>
-          <span>Your Balance 567.89</span>
+          <span>
+            Your Balance{" "}
+            {ethers.utils.formatUnits(tokensMinted.toString(), erc20Decimals)}
```

Comment:
again just want to be sure of what this is, because it might change the number here

Reply:
Same as above.